mldsa/sign.h

```diff
@@ -260,4 +260,47 @@ __contract__(
   ensures(return_value == 0 || return_value == -1)
 );
 
+/*************************************************
```

Comment:
sign.h is not a great place for this. Shall we move it to common.h?

Reply:
This seems to be open still